src/App.js

```diff
@@ -3,7 +3,7 @@ import { Route, Switch } from 'react-router-dom';
 
 import redirectAfterTimeout from './shared/components/hoc/redirect_after_timeout';
 
-import Main from './visitors/pages/main';
+import VisitorSignupPage from './visitors/pages/Signup';
```

Comment:
🙇🏽‍♀️

Reply:
I don't know what this emoji means.

Reply:
![](https://media.giphy.com/media/xUA7aN1MTCZx97V1Ic/giphy.gif)